source/System.cpp

```diff
@@ -538,6 +539,15 @@ void System::UpdateSystem(const Set<System> &systems, const set<double> &neighbo
 
 
 
+void System::UpdateObjectVisibilities(const Ship *flagship)
```

Comment:
This function is currently unused, and we are still discussing if we should run this only on entering a system, or on every frame (with more conditions than only the flagship).

My suggestion would be to leave out this code for now, the code is basically not used in this PR, and when we add modifiers for visibility, then I really hope that we will have more modifiers than only flagship attributes.